ansible/roles/csm.sbps.apply_label/files/apply_k8s_label.sh

```diff
@@ -26,4 +26,9 @@
 LABEL="iscsi=sbps"
 HOST_NAME="$(awk '{print $1}' /etc/hostname)"
 
-kubectl label nodes $HOST_NAME $LABEL
+label_is_present=`kubectl get nodes --show-labels=true $HOST_NAME | grep $LABEL`
+
+if [[ -z $label_is_present ]]
```

Comment:
This can be simplified to
```
if ! kubectl get nodes -l $LABEL | grep -q $HOST_NAME
```
Or the entire check can be eliminated by adding `--overwrite=true` to setting the label:
```
kubectl label nodes --overwrite=true $HOST_NAME $LABEL
```